lib/metrics/eventLoopLag.js

```diff
@@ -1,31 +1,112 @@
 'use strict';
-
 const Gauge = require('../gauge');
 
+// Check if perf_hooks module is available
+let perf_hooks;
+try {
+	// eslint-disable-next-line
```

Comment:
```suggestion
	/* eslint-disable node/no-unsupported-features/node-builtins */
```

makes it more obvious what is being disabled.